tests/sign_verify.rs

```diff
@@ -0,0 +1,80 @@
+use p256_cortex_m4::{PublicKey, SecretKey, Signature};
+
+// message hash
+const HASH: [u8; 32] = [
```

Comment:
Generally, I prefer a dev-dependency on `hex-literal` for these kind of constants. But no worries.